tsfile/src/main/java/org/apache/iotdb/tsfile/file/MetaMarker.java

```diff
@@ -30,6 +30,10 @@
   public static final byte CHUNK_HEADER = 1;
   public static final byte SEPARATOR = 2;
   public static final byte VERSION = 3;
+  // following this marker are two longs marking the minimum and maximum indices of operations
+  // involved in the last flushed MemTable, which are generally used to support checkpoint,
+  // snapshot, or backup.
+  public static final byte OPERATION_INDEX_RANGE = 4;
```

Comment:
Actually in the new tsfile,  we use number 4 as another type of chunk_header. 
https://cwiki.apache.org/confluence/display/IOTDB/New+TsFile+Format
What about change this to 5, or change the new chunk_header marker to 5?